etna/core/tests/test_cookie_tags.py

```diff
@@ -0,0 +1,27 @@
+from django.test import SimpleTestCase
+
+from etna.core.templatetags.cookie_tags import cookie_use_permitted
+
+
+class TestCookieUsePermittedTag(SimpleTestCase):
+
+    empty_cookie = ""
+    none_cookie = None
+    json_usage_true_cookie = '{"usage":true,"settings":false,"essential":true}'
+    json_unicoded_usage_true_cookie = (
+        "%7b%22usage%22%3atrue%2c%22settings%22%3afalse%2c%22essential%22%3atrue%7d"
+    )
+    json_usage_false_cookie = '{"usage":false,"settings":false,"essential":true}'
```

Comment:
Not a bad start, but we should probably try to trigger all of those exceptions we're handling, to make sure it behaves as expected, e.g.
```suggestion
    json_usage_false_cookie = '{"usage":false,"settings":false,"essential":true}'
    invalid_json_cookie = 'NOT_JSON'
    unexpected_json_format = '["item_one", "item_two"]'
```